sdk/spring/spring-cloud-azure-service/src/main/java/com/azure/spring/cloud/service/implementation/identity/impl/StaticAccessTokenCache.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.service.implementation.identity.impl;
+
+import com.azure.core.credential.AccessToken;
+import com.azure.spring.cloud.service.implementation.identity.api.Cache;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Implementation of {@link Cache} that provide static cache.
+ */
+public class StaticAccessTokenCache implements Cache<String, AccessToken> {
+    private static Map<String, AccessToken> cache = new ConcurrentHashMap<>();
```

Comment:
Why do we remove the `final` here?

Reply:
OK, add final will rename `cache` to `CACHE` follow the check-style rules.